quarkchain/cluster/slave.py

```diff
@@ -683,7 +684,15 @@ def close_connection(self, conn):
 
 class SlaveConnection(Connection):
     def __init__(
-        self, env, reader, writer, slave_server, slave_id, chain_mask_list, name=None
+        self,
+        env,
+        reader,
+        writer,
+        slave_server,
+        slave_id,
+        chain_mask_list,
+        type,
```

Comment:
same reason as before. `slave.py` is for QKC slaves, we don't need them to be aware of other types of slaves